ingestion_server/ingestion_server/es_helpers.py

```diff
@@ -45,9 +46,16 @@ def _elasticsearch_connect() -> Elasticsearch:
     :return: an Elasticsearch client
     """
 
+    elasticsearch_scheme = config("ELASTICSEARCH_SCHEME", default="http://")
     elasticsearch_url = config("ELASTICSEARCH_URL", default="localhost")
     elasticsearch_port = config("ELASTICSEARCH_PORT", default=9200, cast=int)
 
+    es_endpoint = "{scheme}{url}:{port}".format(
+        scheme=elasticsearch_scheme,
+        url=elasticsearch_url,
+        port=elasticsearch_port,
+    )
```

Comment:
Why not f-strings? It's been a long time since I've seen `.format` being used.

Reply:
Cleaned up in https://github.com/WordPress/openverse/pull/3029/commits/a4402065a62eb1bcc9a80d38f2b9915b317e6a96